src/main/kotlin/three/consulting/epoc/service/FirebaseService.kt

```diff
@@ -54,5 +54,25 @@ class FirebaseService(
             throw UnableToUpdateEmployeeAndRoleException()
         }
     }
+
+    fun syncAllFirebaseUsers() {
+
+        logger.info { "Starting to sync users found in Firebase with the database" }
+
+        try {
+            val page = FirebaseAuth.getInstance().listUsers(null)
```

Comment:
You already have the firebaseAuth bean defined in the service class constructor, which you could use here. Also `page` is a bad variable name for a list of users